### PR TITLE
SK-1788-WEB-Emails-EmailPro-outlook-android

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+>> ![outlook android email pro](images/outlook-app-android-add-step02.png){.thumbnail .w-400 .h-600}
+>>
+>> > [!warning]
+>> >
+>> > Si la fenêtre de choix du protocole n'apparait pas, appuyez sur le bouton `?` dans le coin supérieur droit de l'écran, puis choisissez `Changer de fournisseur de compte`{.action}. Sélectionnez alors `IMAP`(recommandé) ou `POP3`.<br>
+>> > ![outlook android email pro](images/outlook-app-android-add-step021.png){.thumbnail .w-400 .h-600}
+>>
+> **Etape 3 - IMAP**
+>>
+>> Dans la fenêtre suivante, cochez `Paramètres avancés`{.action} et  complétez les informations suivantes :
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md]:
```suggestion
>> Dans la fenêtre suivante, cochez `Paramètres avancés`{.action} et complétez les informations suivantes :
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+>> - **Serveur de courrier entrant IMAP** :<br>- **Nom d'hôte IMAP** : saisissez `pro**?**.mail.ovh.net` (remplacez bien «**?**» par le numéro de votre serveur).<br>- **Port** : 993<br>- **Type de sécurité** : SSL/TLS<br>- **Nom d'utilisateur IMAP** : votre adresse e-mail complète<br>- **Mot de passe IMAP** : celui de votre adresse e-mail
+>> - **Serveur de courrier entrant SMTP** :<br>- **Nom d'hôte SMTP** : saisissez `pro**?**.mail.ovh.net` (remplacez bien «**?**» par le numéro de votre serveur).<br>- **Port** : 587<br>- **Type de sécurité** : SSL/TLS<br>- **Nom d'utilisateur SMTP** : votre adresse e-mail complète<br>- **Mot de passe SMTP** : celui de votre adresse e-mail
+>>
+>> Pour finaliser la configuration, cliquez sur le bouton « &#10003; »
+>>
+>> ![outlook android email pro](images/outlook-app-android-add-step03-imap-eu.png){.thumbnail .w-400 .h-600}
+>>
+> **Etape 3 - POP3**
+>>
+>> Dans la fenêtre suivante, cochez `Paramètres avancés`{.action} et  complétez les informations suivantes :
```
Comment (Kazarus4) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md]:
```suggestion
>> Dans la fenêtre suivante, cochez `Paramètres avancés`{.action} et complétez les informations suivantes :
```

**Review by Kazarus4 — COMMENTED** [review]
FR review

**Review by Kazarus4 — COMMENTED** [review]
FR review

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+OVHcloud propose aussi une application web permettant d'accéder à votre adresse e-mail depuis un navigateur internet. Celle-ci est accessible via ce lien : [Webmail](/links/web/email). Vous pouvez vous y connecter grâce aux identifiants de votre adresse e-mail. Pour toute question relative à son utilisation, aidez-vous de notre guide [Consulter son compte depuis l’interface OWA](/pages/web_cloud/email_and_collaborative_solutions/using_the_outlook_web_app_webmail/email_owa).
+
+### Modifier les paramètres existants <a name="modify-settings"></a>
+
+L'application Outlook ne permet pas de modifier les paramètres serveur de votre compte e-mail.
+
+Si votre compte e-mail est déjà paramétré et que vous souhaitez le paramétrer à nouveau, vous devez alors le supprimer et le recréer :
+
+1. Appuyez sur l'enveloppe « &#9993; » dans la partie supérieure gauche de votre écran.
+2. Appuyez sur l'icöne de réglage « &#9965; » dans le bas de la colonne de gauche.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md]:
```suggestion
2. Appuyez sur l'icône de réglage « &#9965; » dans le bas de la colonne de gauche.
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+3. Dans la section « Général » appuyez sur `Comptes` pour visualiser l'ensemble des adresses e-mail configurées sur l'application.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md]:
```suggestion
3. Dans la section « Général », appuyez sur `Comptes` pour visualiser l'ensemble des adresses e-mail configurées sur l'application.
```

File: pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+- Sélectionnez le compte e-mail concerné.
+- Appuyez sur `Supprimer le compte`{.action}.
+- Appuyez sur `Supprimer`{.action} à la question  « Voulez-vous supprimer le compte ? ».
+
+![outlook android email pro](images/outlook-app-android-delete-account-02.png){.thumbnail .w-400 .h-600}
+
+> [!success]
+>
+> Une fois votre compte e-mail supprimé, suivez les instructions de la partie « [Ajouter le compte](#add-account) » sur cette documentation.
```
Comment (Y0Coss) [inline pages/web_cloud/email_and_collaborative_solutions/email_pro/emailpro_outlook_app_android/guide.fr-fr.md]:
```suggestion
> Une fois votre compte e-mail supprimé, suivez les instructions de la partie « [Ajouter le compte](#add-account) » de cette documentation.
```

**Review by Y0Coss — COMMENTED** [review]
The index needs to be edited

**Pull request merged**